vllm/model_executor/layers/fused_moe/rocm_aiter_fused_moe.py

```diff
@@ -274,6 +140,50 @@ def rocm_aiter_biased_grouped_topk_fake(
     pass
 
 
+def rocm_aiter_fused_moe_impl(
+    hidden_states: torch.Tensor,
+    w1: torch.Tensor,
+    w2: torch.Tensor,
+    topk_weight: torch.Tensor,
+    topk_ids: torch.Tensor,
+    expert_mask: Optional[torch.Tensor] = None,
+    activation_method: int = ActivationMethod.SILU.value,
+    quant_method: int = QuantMethod.NO.value,
```

Comment:
Why not require the enum to be passed in here, and then do `.value` inside the function?
```suggestion
    activation_method: ActivationMethod = ActivationMethod.SILU,
    quant_method: QuantMethod = QuantMethod.NO,
```

Reply:
@ProExpertProg unfortunately we can't use the Enum type directly as the `direct_register_custom_op` complains about Enum type argument and does not register the function in torch schema.